examples/ex10p.cpp

```diff
@@ -178,6 +178,7 @@ int main(int argc, char *argv[])
    double visc = 1e-2;
    double mu = 0.25;
    double K = 5.0;
+   bool adaptive_lin_rtol = true;
```

Comment:
It looks like this parameter is not used. Also, why not leave it `false` by default?

Reply:
Also, should we add this option in the serial `ex10`?

Reply:
Good points